variable.tf

```diff
@@ -48,10 +48,17 @@ variable "az" {
   default = "eu-west-2a"
 }
 
-variable "instance_type" {
+variable "namespace" {
```

Comment:
What is this variable `namespace`?
It doesn't appear to be populated.

Reply:
The namespace is populated through dq-tf-infra. see https://github.com/UKHomeOffice/dq-tf-infra/blob/master/ops.tf#L38